dev/com.ibm.ws.org.eclipse.yasson.2.0/resources/META-INF/services/jakarta.json.bind.spi.JsonbProvider

```diff
@@ -0,0 +1 @@
+org.eclipse.yasson.JsonBindingProvider
```

Comment:
This implies the transformer tool isn't renaming `META-INF/service/javax.*` files to their `jakarta.*`, is there an issue open on the transformer tool to track that?

Reply:
Yes.  I opened issue https://github.com/tbitonti/jakartaee-prototype/issues/97 to document and track this issue.